File: _replace_links.js
const fs = require('fs');
const files = [
    'contact-us.html',
    'exhibits.html',
    'history.html',
    'mandate-objectives.html',
    'news-events.html',
    'vision-mission.html'
];

for (const f of files) {
    let c = fs.readFileSync(f, 'utf8');
    c = c.replace(/href="paf-hero\.html"/g, 'href="index.html"');
    fs.writeFileSync(f, c, 'utf8');
    console.log('Done:', f);
}
console.log('All done!');

File: _replace_links.py
import re

files = [
    'contact-us.html',
    'exhibits.html',
    'history.html',
    'mandate-objectives.html',
    'news-events.html',
    'vision-mission.html'
]

for f in files:
    with open(f, 'r', encoding='utf-8') as fh:
        c = fh.read()
    c = re.sub(r'href="paf-hero\.html"', 'href="index.html"', c)
    with open(f, 'w', encoding='utf-8') as fh:
        fh.write(c)
    print(f'Done: {f}')
print('All done!')
